internal/connect/command_cluster_create.go

```diff
@@ -25,13 +25,44 @@ func (c *clusterCommand) newCreateCommand() *cobra.Command {
 		RunE:        c.create,
 		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
 		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "The configuration file contains configs including offsets for the connector.",
+				Code: `{
```

Comment:
Code inside example is different from other examples, is it expected here?

Reply:
Yes, kept in consistent with update offset.